interface-definitions/include/dhcp/ddns-settings.xml.i

```diff
@@ -0,0 +1,97 @@
+<!-- include start from dhcp/ddns-settings.xml.i -->
+<leafNode name="send-updates">
+<properties>
+    <help>Send updates for this scope</help>
+    <valueless/>
+</properties>
+</leafNode>
+<leafNode name="override-client-update">
+<properties>
+    <help>Always update both forward and reverse DNS data, regardless of the client's request</help>
+    <valueless/>
+</properties>
+</leafNode>
+<leafNode name="override-no-update">
+<properties>
+    <help>Perform a DDNS update, even if the client instructs the server not to</help>
+    <valueless/>
+</properties>
+</leafNode>
+<leafNode name="replace-client-name">
+<properties>
+    <help>Replace client name mode</help>
+    <completionHelp>
+    <list>never always when-present when-not-present</list>
```

Comment:
Could you please indent every nesting level? Unindented nested tags are quite hard to read.